ipset.go

```diff
@@ -99,6 +112,23 @@ func (s *IPSet) RemoveSet(b *IPSet) {
 	}
 }
 
+// Invert updates s to contain the opposite of is current contents.
```

Comment:
Consider calling this `Complement`, since that's the standard set theory name for it.

s/is/its/

Reply:
Doh, I knew I was using the wrong word. Thanks!